models/DataObject/ClassDefinition/Data/InputQuantityValue.php

```diff
@@ -161,4 +127,22 @@ public function denormalize(mixed $value, array $params = []): ?InputQuantityVal
 
         return null;
     }
+
+    public function getColumnType(): array|string|null
+    {
+        return [
+            'value' => 'varchar(255)',
+            'unit' => 'varchar(50)',
+        ];
+    }
+
+    public function getQueryColumnType(): array|string|null
+    {
+        return $this->getColumnType();
```

Comment:
I noticed the call to `getColumnType` here, which is fine, because column types and query column types are the same. But at other instances we just copied the definition, e.g. https://github.com/pimcore/pimcore/pull/13888/files#diff-f9e8614c6fe54e4873af1506ee66172b2c8aad0ee906e85e93b07c8905382c53R439 and/or https://github.com/pimcore/pimcore/pull/13888/files#diff-f9e8614c6fe54e4873af1506ee66172b2c8aad0ee906e85e93b07c8905382c53R444

Maybe we should be consistent here?